app/bundles/DynamicContentBundle/EventListener/StatsSubscriber.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\DynamicContentBundle\EventListener;
+
+use Doctrine\ORM\EntityManager;
+use Mautic\CoreBundle\EventListener\StatsSubscriber as CommonStatsSubscriber;
+
+/**
+ * Class StatsSubscriber.
+ */
+class StatsSubscriber extends CommonStatsSubscriber
+{
+    /**
+     * StatsSubscriber constructor.
+     *
+     * @param EntityManager $em
+     */
+    public function __construct(EntityManager $em)
+    {
+        // ??? Attempted to call an undefined method named "loadMetadata" of class "Mautic\DynamicContentBundle\Entity\StatRepository". ???
```

Comment:
Is this something that needs to be addressed? It could be that we aren't collecting stats on DWC.